var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -31,10 +31,10 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.3', 'a5f65570cd9107ec8a8ec88f17b31bb1')
```

Comment:
If you read https://github.com/westes/flex/issues/113, you'll see that 2.6.2 had serious flaws that broke backwards compatibility. This was fixed in 2.6.3, so we just commented out 2.6.2 with a note. Before we realized that this was a flex problem and not a "everything that depends on flex" problem, we must have added `depends_on('flex@:2.6.1')` to at least one package. This can be removed now. The real problem is a bug in the concretization algorithm though.